app/views/admin/organisations/index.html.erb

```diff
@@ -1,6 +1,13 @@
 <h1 class="govuk-heading-l">All organisations</h1>
-<h3 class="govuk-heading-m">
-  <%= pluralize(@organisations.count, 'organisation') %> are signed up to GovWifi
+<h3 class="govuk-body">
+  GovWifi is in
+  <span class="govuk-!-font-weight-bold">
+    <%= pluralize(@location_count, 'location') %>
+  </span>
+   across
+  <span class="govuk-!-font-weight-bold">
+    <%= pluralize(@organisations.count, 'organisation') %>.
```

Comment:
Would `@organisations.length` be more performant here, as the `@organisations` is already loaded, calling `.length` on it as if its an array means it doesn't run the SQL again.

Reply:
'scuse me missing this same comment on the other PR! I'll try it out - hadn't realised `.count` would trigger another query.

Reply:
Yep, that works, cuts out a query - nice one.

Reply:
Addressed in https://github.com/alphagov/govwifi-admin/pull/503/commits/b7905fbb5a31e77e6902d6a2eed561b6c532d116